stable/grafana/values.yaml

```diff
@@ -154,6 +153,35 @@ persistence:
   # subPath: ""
   # existingClaim:
 
+initChownData:
+  ## If false, data ownership will not be reset at startup
+  ## This allows the prometheus-server to be run with an arbitrary user
+  ##
+  enabled: true
+
+  ## initChownData container name
+  ##
+  name: init-chown-data
+
+  ## initChownData container image
+  ##
+  image:
+    repository: busybox
+    tag: latest
+    pullPolicy: IfNotPresent
+
+  ## initChownData resource requests and limits
+  ## Ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ##
+  resources: {}
```

Comment:
yay!

Reply:
What do you mine? There we will configure the resources of the pods in case is needed

Reply:
i just meant that this is the thing that is behind this entire PR, so i am happy to see it!

Reply:
hehe. It's nice to see your happiness :)